packages/block-library/src/navigation-link/edit.js

```diff
@@ -320,12 +359,19 @@ export default compose( [
 			!! navigationBlockAttributes.showSubmenuIcon && hasDescendants;
 		const isParentOfSelectedBlock = hasSelectedInnerBlock( clientId, true );
 
+		const userCanCreatePages = select( 'core' ).canUser(
+			'create',
+			'pages'
+		);
+
 		return {
 			isParentOfSelectedBlock,
 			hasDescendants,
 			showSubmenuIcon,
 			textColor: navigationBlockAttributes.textColor,
 			backgroundColor: navigationBlockAttributes.backgroundColor,
+			navigationBlockAttributes,
```

Comment:
Where is this being used in the component?

Reply:
Removed. Rebase error...again. Sorry.